app/controllers/user.controller.js

```diff
@@ -244,3 +244,84 @@ exports.deleteAll = (req, res) => {
       });
     });
 };
+
+
+// Reset password
+
+exports.resetPassword = async (req, res) => {
+
+  
+  if (req.body.otp === undefined) {
+    const error = new Error("OTP cannot be empty for !");
+    error.statusCode = 400;
+    res.status(400).send({
```

Comment:
before sending a response back to the frontend, assuming it will capture and log the error, you should `console.log` the problem (and relevant context) here (and line 264 and line 273 ...)